openedx/core/djangoapps/user_api/accounts/tests/pact/verify_user.py

```diff
@@ -0,0 +1,79 @@
+"""
+User Verification Server for Profile Information
+"""
+import os
+import logging
+from django.test import LiveServerTestCase
+from django.urls import reverse
+from pact import Verifier
+
+from common.djangoapps.student.tests.factories import UserFactory
+from common.djangoapps.student.models import User
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_POST
+import json
+
+log = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+PACT_DIR = os.path.dirname(os.path.realpath(__file__))
+PACT_FILE = "frontend-app-profile-edx-platform.json"
+
+
+class ProviderState():
+    """ Provider State for the testing profile """
+
+    def account_setup(self, request):
+        """ Sets up the Profile that we want to mock in accordance to our contract """
+        User.objects.filter(username="staff").delete()
+        user_acc = UserFactory.create(username="staff")
```

Comment:
This will end up deleting and recreating the `staff` user that devstack is provisioned with, right? I think we'd be better off using a pact-specific username that won't alter the standard set of accounts people use for local testing.

Reply:
Would something like `pact_staff` be more appropriate?

Reply:
Yeah, anything with pact in the name is probably fine.